js/majors.js

```diff
@@ -746,9 +746,12 @@ window.majors = {
             "on the EECS website and 6.AcAd on Piazza.",
         skip: 1
       },
-      [1, [2, [1, "6.01", "6.S08"],
+      [1, [2, "6.0001",
+              [ {count:1, globalMatchesIgnore:1},
+                "6.01", "6.02", "6.03", "6.S08"] ]
```

Comment:
You need to add the trailing comma here, though. I pulled your commit to test and this threw an error.